frontend/src/metabase/App.jsx

```diff
@@ -134,13 +142,58 @@ export default class App extends Component {
             />
           </SearchBarContent>
         </SearchBarContainer>
-        <ProfileLinkContainer>
-          <ProfileLink {...this.props} user={currentUser} />
-        </ProfileLinkContainer>
+        <NewButton setModal={this.setModal.bind(this)} />
```

Comment:
Let's convert `setModal` into an arrow function, so it's bound automatically. Like:

```js
setModal = modal => {
   this.setState({ modal })
   // popover stuff
}
```